src/core_processor/camera_calibration/camera_calibrator.py

```diff
@@ -0,0 +1,24 @@
+import logging
+from dataclasses import dataclass
+
+import numpy as np
+
+from src.cameras.capture.opencv_camera.opencv_camera import OpenCVCamera
+from src.core_processor.camera_calibration.lens_distortion_calibrator import LensDistortionCalibrator, \
+    LensDistortionCalibrationData
+
+logger = logging.getLogger(__name__)
+
+
+class CameraCalibrator:
+    def __init__(self):
+        self.lens_distortion_calibrator = LensDistortionCalibrator()
+
+    def calibrate(self, this_camera: OpenCVCamera):
+        if self.lens_distortion_calibrator is None:
```

Comment:
why would this ever be `None`? We definitely want to avoid "setup" kinds of workflows within "work" workflows to keep the code simple.

Reply:
fixed